ext/storage_sources/azure_store/azure_connection.cpp

```diff
@@ -31,16 +31,37 @@
 #include <azure/core.hpp>
 #include <azure/storage/blobs.hpp>
 
+#include <cstdlib>
+#include <iostream>
+#include <vector>
+#include <string>
```

Comment:
Can we clean the includes here? It seems some are already included in the .h file (e.g. strin) and we aren't using iostream, or maybe ctsdlib. I think vector might be already included in the .h file somewhere as well.

Reply:
yep done :)